### DISCUSSION TITLE
how to change the conversation_max_tokens?

### DISCUSSION THREAD

**Discussion by Aurthoria** [ddiu8081/chatgpt-demo#421] [Q&A]
just the Title

**Comment by ddiu8081** [comment]
Currently it's not possible to adjust this option in the front-end way for now (You can do this in [anse](https://github.com/anse-app/anse)). 

But you can do this by modifying the code in [`src/utils/OpenAI.ts`](https://github.com/anse-app/chatgpt-demo/blob/main/src/utils/openAI.ts):

https://github.com/anse-app/chatgpt-demo/blob/5fd4cadc7faca753e765b5567170abe17cc40f97/src/utils/openAI.ts#L7-L19
